src/components/Tasks/TaskList.tsx

```diff
@@ -37,10 +35,10 @@ interface PropsType {
 }
 
 export default function TaskList(props: PropsType) {
-  const [showCompleted, setShowCompleted] = React.useState(false);
   const settings = useSelector((state: StoreState) => state.settings.taskSettings);
   const { filterBy } = settings;
   const theme = useTheme();
+  const [showCompleted, setShowCompleted] = React.useState(false);
```

Comment:
Please move it back to here it was (look at the diff of the change as a whole). It's just an unrelated change now.